src/controllers/cohort.js

```diff
@@ -2,11 +2,37 @@ import { createCohort } from '../domain/cohort.js'
 import { sendDataResponse, sendMessageResponse } from '../utils/responses.js'
 
 export const create = async (req, res) => {
+  const parseAndValidateDate = (dateString) => {
+    const regex = /^\d{4}\/\d{2}\/\d{2}$/
```

Comment:
let's get a better name for this variable that describes what the matcher is doing.